Examples/Io/Root/src/VertexNTupleWriter.cpp

```diff
@@ -538,32 +538,34 @@ ProcessCode VertexNTupleWriter::writeT(
     auto& recoToTruth = recoToTruthMatching.back();
 
     // We have to decide if this reco vertex is a split vertex.
-    if (auto it = truthToRecoMatching.find(truthMajorityVertexId);
-        it != truthToRecoMatching.end()) {
-      // This truth vertex is already matched to a reconstructed vertex so we
-      // are dealing with a split vertex.
+    if (truthMajorityVertexId.has_value()) {
```

Comment:
nitpick: nesting can be avoided with `if (!bla) { continue; }` above

Reply:
something like

```suggestion
    if (!truthMajorityVertexId.has_value()) {
      // No truth vertex matched to this reconstructed vertex
      ACTS_DEBUG("No truth vertex matched to this reconstructed vertex.");
      continue;
    }

```

Reply:
Changed now